src/eko/io/runcards.py

```diff
@@ -272,6 +274,21 @@ def update(theory: Union[RawCard, TheoryCard], operator: Union[RawCard, Operator
     return cards.new_theory, cards.new_operator
 
 
+def default_atlas(masses: list, matching_ratios: list):
+    r"""Create default landscape.
+
+    This method should not be used to write new runcards, but rather to have a
+    consistent default for comparison with other softwares and existing PDF
+    sets.
+    There is no one-to-one relation between number of running flavors and final
+    scales, unless matchings are all applied. But this is a custom choice,
+    since it is possible to have PDFs in different |FNS| at the same scales.
```

Comment:
This is copy-pasted from below :P

I'd propose to drop this, and to prepend an underscore to this function.
At some point, I will also go through #175 and reconsider these objects organically.